2009-04-09  Michael Tautschnig  <mt@debian.org>

	* get-config-dir-git: Do not clone into an existing directory (thanks
		Sebastian Harl) (closes: #506463)
Index: trunk/lib/get-config-dir-git
===================================================================
--- trunk.orig/lib/get-config-dir-git
+++ trunk/lib/get-config-dir-git	
@@ -35,6 +35,8 @@
    task_error 701 $?
 else 
    echo "Checking out from git"
+   # cloning into an existing directory is not allowed
+   if [ -d $FAI ]; then rmdir $FAI; fi
    git clone $giturl $FAI 
    task_error 702 $?
 fi
